specification/app/resource-manager/Microsoft.App/preview/2023-04-01-preview/CommonDefinitions.json

```diff
@@ -1149,6 +1159,20 @@
           ]
         }
       }
+    },
+    "ServiceBind": {
+      "description": "Configuration to bind a ContainerApp to a dev ContainerApp Service",
```

Comment:
More explanation to `dev ContainerApp Service`

Reply:
Will leave that to docs for now as the details will get pretty verbose here. Though the final docs links are not yet available.